osx/post_installation.sh

```diff
@@ -124,7 +124,19 @@ if [ ! -d "$LAUNCH_AGENTS" ]; then
     fi
 fi
 
+
+((STEP++))
+echo "$STEP/$STEPS. Create plist in /Library/LaunchAgents folders..."
+if [ ! -d "$LAUNCH_AGENTS" ]; then
+    echo ".. Must create '$LAUNCH_AGENTS' folder..."
+    sudo mkdir -p $LAUNCH_AGENTS
+    if [ $? -ne 0 ]; then
+        echo ".. Abort!  Error encountered creating '$LAUNCH_AGENTS' directory."
+        exit 1
+    fi
+fi
 create_plist
+sudo cp $PLIST_SRC $LAUNCH_AGENTS
```

Comment:
This one too.  If you want to test, just run with `sudo ./post_installation.sh` @amodia.